cmd/webapi/main.go

```diff
@@ -10,7 +10,8 @@ import (
 func main() {
```

Comment:
What's the aim of keeping a main for webapi now's it's been merged into control?

And won't it crash on it's first request due to the db being nil?

Reply:
Oh, yeah, this can be deleted (groundstation too)